cmd/server/main.go

```diff
@@ -308,6 +308,7 @@ func realMain(ctx context.Context) error {
 
 		userController := user.New(ctx, cacher, config, db, h)
 		userSub.Handle("", userController.HandleIndex()).Methods("GET")
+		userSub.Handle("", userController.HandleIndex()).Queries("offset", "{[0-9]*?}").Methods("GET")
```

Comment:
I think this might be better as:

```go
userSub.Handle("page/{offset:[0-9]+}", usersController.HandleIndex()).Methods("GET")
```

And then pull the pagination from `mux.Vars()`. That will give us nicer urls like `/users/page/2`